app/src/main/res/values/strings.xml

```diff
@@ -102,10 +102,10 @@
     <string name="entry_otp">OTP</string>
     <string name="entry_url">URL</string>
     <string name="entry_user_name">Username</string>
-    <string name="error_arc4">The Arcfour stream cipher is not supported.</string>
+    <string name="error_arc4">The RC4 stream cipher is not supported.</string>
```

Comment:
The name RC4 is trademarked, so RC4 is often referred to as ARCFOUR or ARC4 (meaning alleged RC4)[12] to avoid trademark problems. RSA Security has never officially released the algorithm;